benches/signal.rs

```diff
@@ -45,9 +45,9 @@ fn many_signals(bench: &mut Bencher) {
     let num_signals = 10;
     let (tx, mut rx) = mpsc::channel(num_signals);
 
-    let rt = runtime::Builder::new()
+    let rt = runtime::Builder::new_multi_thread()
         // Intentionally single threaded to measure delays in propagating wakes
-        .basic_scheduler()
+        .worker_threads(0)
```

Comment:
This to me doesn't make sense, why 0? I would imagine we should not allow that unless the multi thread runtime can be a current thread one?